features/core-download.feature

```diff
@@ -14,6 +14,11 @@ Feature: Download WordPress
     And the {SUITE_CACHE_DIR}/core/wordpress-{VERSION}-en_US.tar.gz file should exist
     And the {SUITE_CACHE_DIR}/core/wordpress-{WP_VERSION-latest}-en_US.tar.gz file should exist
 
+    When I run `wp core download --skip-content`
+    And save STDOUT 'Downloading WordPress ([\d\.]+)' as {VERSION}
+    Then the wp-settings.php file should exist
+    And the {SUITE_CACHE_DIR}/core/wordpress-{WP_VERSION-latest}-without-content-en_US.zip file should exist
```

Comment:
Can you create a new Scenario for this behavior please (instead of repurposing an existing one)?

Reply:
We'll also need to make sure there are tests for:

* Non US locales
* Use of `--version=<version>`

Reply:
Thanks @danielbachhuber, I've just created 3 new scenarios.